### PR TITLE
Provide QueryLimit options for Third Party Logs API (Loki)

### PR DESCRIPTION
**Pull request by khrm**
This adds a new config to provide a query limit for Third Party Logging API.

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you review them:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] [Tested your changes locally](https://github.com/tektoncd/results/blob/main/docs/DEVELOPMENT.md) (if this is a code change)
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [ ] Has a kind label. You can add a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user-facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contain the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
This adds a new config to provide a query limit for Third Party Logging API.: LOGGING_PLUGIN_QUERY_LIMIT
```

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/assign @vdemeester @enarha

**Review by vdemeester — APPROVED** [review]

**Comment by khrm** [comment]
@enarha Yes. By default, the limit is only 100.

File: docs/logging-support.md [hunk 1]
```diff
@@ -28,3 +28,4 @@ These are the common configuration options for all third party logging APIs.
 - `LOGGING_PLUGIN_CA_CERT`: The CA certificate to use for the third party logging API. This should ideally be passed as environment variable in the deployment spec of the results-api pod. (optional)
 - `LOGGING_PLUGIN_TLS_VERIFICATION_DISABLE`: Set to `true` to disable TLS verification for the third party logging API. (optional)
 - `LOGGING_PLUGIN_FORWARDER_DELAY_DURATION`: This is the max duration in minutes taken by third party logging system to forward and store the logs after completion of taskrun and pipelinerun. This is used to search between start time of runs and completion plus buffer duration.
+- `LOGGING_PLUGIN_QUERY_LIMIT`: This sets the query limit for Third Party Logging API if logging backend has a limit on number of logs lines.
```
Comment (enarha) [inline docs/logging-support.md:31]: Is that optional? If yes, maybe mark it as such e.g (optional).

```suggestion
- `LOGGING_PLUGIN_QUERY_LIMIT`: Sets the query limit for Third Party Logging API if logging backend has a limit on number of log lines returned.
```

**Comment by khrm** [comment]
Changed the documentation, please relook. @enarha

**Review by enarha — COMMENTED** [review]
/lgtm

**Pull request merged**